examples/scf/lcao_Si2/README

```diff
@@ -1,6 +1,6 @@
 /*******************************************************************************/
 /
-/ This is the examples of ABACUS program.
+/ These are the examples of ABACUS program.
 /
 /*******************************************************************************/
```

Comment:
I suggest removing this part in test cases, only preserved in examples/Readme.

Reply:
I think this is just a title, a unified format

Reply:
I think it is ok to keep it.